### ISSUE TITLE
Auto complete not working since 8.0 update

### ISSUE THREAD

**Issue by jpdery** [CodinGame/monaco-editor-react#53]
I've recently updated to the 8.0 release and the auto complete feature is not working anymore in HTML, CSS and JavaScript.

I'm loading the following extensions:

```
import '@codingame/monaco-editor-wrapper/features/extensionHostWorker'
import '@codingame/monaco-vscode-css-language-features-default-extension'
import '@codingame/monaco-vscode-html-language-features-default-extension'
import '@codingame/monaco-vscode-typescript-language-features-default-extension'
```

I've also noticed the version of the `@codingame/monaco-editor-wrapper` dependency was a bit behind, currently set to 8.0.0 so I also tried by manually setting it to 8.2.0 but it didn't change anything.

Thanks

**Comment by CGNonofr** [comment]
What version were you on using before the update?

Can you confirm you don't have duplicated dependency versions by running `npm list vscode` and paste the result here?

> I've also noticed the version of the @codingame/monaco-editor-wrapper dependency was a bit behind, currently set to 8.0.0 so I also tried by manually setting it to 8.2.0 but it didn't change anything.

The version is `^8.0.0` which accepts any minor version though

**Comment by jpdery** [comment]
I was on version 7 for monaco-editor-react and 6 for monaco-vs-code-css/html/typescript-extension.

Here is the result for the command npm list vscode

`├─┬ @codingame/monaco-editor-react@8.0.0
│ ├─┬ @codingame/monaco-editor-wrapper@8.2.0
│ │ ├─┬ @codingame/monaco-vscode-accessibility-service-override@7.1.1
│ │ │ └── vscode@npm:@codingame/monaco-vscode-api@7.1.1 deduped
│ │ ├─┬ @codingame/monaco-vscode-clojure-default-extension@7.1.1
│ │ │ └── vscode@npm:@codingame/monaco-vscode-api@7.1.1 deduped
│ │ ├─┬ @codingame/monaco-vscode-coffeescript-default-extension@7.1.1
│ │ │ └── vscode@npm:@codingame/monaco-vscode-api@7.1.1 deduped
│ │ ├─┬ @codingame/monaco-vscode-configuration-editing-default-extension@7.1.1
│ │ │ └── vscode@npm:@codingame/monaco-vscode-api@7.1.1 deduped
│ │ ├─┬ @codingame/monaco-vscode-configuration-service-override@7.1.1
│ │ │ └── vscode@npm:@codingame/monaco-vscode-api@7.1.1 deduped
│ │ ├─┬ @codingame/monaco-vscode-cpp-default-extension@7.1.1
│ │ │ └── vscode@npm:@codingame/monaco-vscode-api@7.1.1 deduped
│ │ ├─┬ @codingame/monaco-vscode-csharp-default-extension@7.1.1
│ │ │ └── vscode@npm:@codingame/monaco-vscode-api@7.1.1 deduped
│ │ ├─┬ @codingame/monaco-vscode-css-default-extension@7.1.1
│ │ │ └── vscode@npm:@codingame/monaco-vscode-api@7.1.1 deduped
│ │ ├─┬ @codingame/monaco-vscode-debug-service-override@7.1.1
│ │ │ └── vscode@npm:@codingame/monaco-vscode-api@7.1.1 deduped
│ │ ├─┬ @codingame/monaco-vscode-dialogs-service-override@7.1.1
│ │ │ └── vscode@npm:@codingame/monaco-vscode-api@7.1.1 deduped
│ │ ├─┬ @codingame/monaco-vscode-diff-default-extension@7.1.1
│ │ │ └── vscode@npm:@codingame/monaco-vscode-api@7.1.1 deduped
│ │ ├─┬ @codingame/monaco-vscode-editor-service-override@7.1.1
│ │ │ └── vscode@npm:@codingame/monaco-vscode-api@7.1.1 deduped
│ │ ├─┬ @codingame/monaco-vscode-emmet-service-override@7.1.1
│ │ │ └── vscode@npm:@codingame/monaco-vscode-api@7.1.1 deduped
│ │ ├─┬ @codingame/monaco-vscode-explorer-service-override@7.1.1
│ │ │ └── vscode@npm:@codingame/monaco-vscode-api@7.1.1 deduped
│ │ ├─┬ @codingame/monaco-vscode-extension-gallery-service-override@7.1.1
│ │ │ └── vscode@npm:@codingame/monaco-vscode-api@7.1.1 deduped
│ │ ├─┬ @codingame/monaco-vscode-extensions-service-override@7.1.1
│ │ │ └── vscode@npm:@codingame/monaco-vscode-api@7.1.1 deduped
│ │ ├─┬ @codingame/monaco-vscode-files-service-override@7.1.1
│ │ │ └── vscode@npm:@codingame/monaco-vscode-api@7.1.1 deduped
│ │ ├─┬ @codingame/monaco-vscode-fsharp-default-extension@7.1.1
│ │ │ └── vscode@npm:@codingame/monaco-vscode-api@7.1.1 deduped
│ │ ├─┬ @codingame/monaco-vscode-go-default-extension@7.1.1
│ │ │ └── vscode@npm:@codingame/monaco-vscode-api@7.1.1 deduped
│ │ ├─┬ @codingame/monaco-vscode-groovy-default-extension@7.1.1
│ │ │ └── vscode@npm:@codingame/monaco-vscode-api@7.1.1 deduped
│ │ ├─┬ @codingame/monaco-vscode-html-default-extension@7.1.1
│ │ │ └── vscode@npm:@codingame/monaco-vscode-api@7.1.1 deduped
│ │ ├─┬ @codingame/monaco-vscode-java-default-extension@7.1.1
│ │ │ └── vscode@npm:@codingame/monaco-vscode-api@7.1.1 deduped
│ │ ├─┬ @codingame/monaco-vscode-javascript-default-extension@7.1.1
│ │ │ └── vscode@npm:@codingame/monaco-vscode-api@7.1.1 deduped
│ │ ├─┬ @codingame/monaco-vscode-json-default-extension@7.1.1
│ │ │ └── vscode@npm:@codingame/monaco-vscode-api@7.1.1 deduped
│ │ ├─┬ @codingame/monaco-vscode-julia-default-extension@7.1.1
│ │ │ └── vscode@npm:@codingame/monaco-vscode-api@7.1.1 deduped
│ │ ├─┬ @codingame/monaco-vscode-keybindings-service-override@7.1.1
│ │ │ └── vscode@npm:@codingame/monaco-vscode-api@7.1.1 deduped
│ │ ├─┬ @codingame/monaco-vscode-language-detection-worker-service-override@7.1.1
│ │ │ └── vscode@npm:@codingame/monaco-vscode-api@7.1.1 deduped
│ │ ├─┬ @codingame/monaco-vscode-languages-service-override@7.1.1
│ │ │ └── vscode@npm:@codingame/monaco-vscode-api@7.1.1 deduped
│ │ ├─┬ @codingame/monaco-vscode-lifecycle-service-override@7.1.1
│ │ │ └── vscode@npm:@codingame/monaco-vscode-api@7.1.1 deduped
│ │ ├─┬ @codingame/monaco-vscode-log-service-override@7.1.1
│ │ │ └── vscode@npm:@codingame/monaco-vscode-api@7.1.1 deduped
│ │ ├─┬ @codingame/monaco-vscode-lua-default-extension@7.1.1
│ │ │ └── vscode@npm:@codingame/monaco-vscode-api@7.1.1 deduped
│ │ ├─┬ @codingame/monaco-vscode-markdown-basics-default-extension@7.1.1
│ │ │ └── vscode@npm:@codingame/monaco-vscode-api@7.1.1 deduped
│ │ ├─┬ @codingame/monaco-vscode-markdown-language-features-default-extension@7.1.1
│ │ │ └── vscode@npm:@codingame/monaco-vscode-api@7.1.1 deduped
│ │ ├─┬ @codingame/monaco-vscode-markdown-math-default-extension@7.1.1
│ │ │ └── vscode@npm:@codingame/monaco-vscode-api@7.1.1 deduped
│ │ ├─┬ @codingame/monaco-vscode-markers-service-override@7.1.1
│ │ │ └── vscode@npm:@codingame/monaco-vscode-api@7.1.1 deduped
│ │ ├─┬ @codingame/monaco-vscode-media-preview-default-extension@7.1.1
│ │ │ └── vscode@npm:@codingame/monaco-vscode-api@7.1.1 deduped
│ │ ├─┬ @codingame/monaco-vscode-model-service-override@7.1.1
│ │ │ └── vscode@npm:@codingame/monaco-vscode-api@7.1.1 deduped
│ │ ├─┬ @codingame/monaco-vscode-multi-diff-editor-service-override@7.1.1
│ │ │ └── vscode@npm:@codingame/monaco-vscode-api@7.1.1 deduped
│ │ ├─┬ @codingame/monaco-vscode-notifications-service-override@7.1.1
│ │ │ └── vscode@npm:@codingame/monaco-vscode-api@7.1.1 deduped
│ │ ├─┬ @codingame/monaco-vscode-npm-default-extension@7.1.1
│ │ │ └── vscode@npm:@codingame/monaco-vscode-api@7.1.1 deduped
│ │ ├─┬ @codingame/monaco-vscode-objective-c-default-extension@7.1.1
│ │ │ └── vscode@npm:@codingame/monaco-vscode-api@7.1.1 deduped
│ │ ├─┬ @codingame/monaco-vscode-outline-service-override@7.1.1
│ │ │ └── vscode@npm:@codingame/monaco-vscode-api@7.1.1 deduped
│ │ ├─┬ @codingame/monaco-vscode-output-service-override@7.1.1
│ │ │ └── vscode@npm:@codingame/monaco-vscode-api@7.1.1 deduped
│ │ ├─┬ @codingame/monaco-vscode-perl-default-extension@7.1.1
│ │ │ └── vscode@npm:@codingame/monaco-vscode-api@7.1.1 deduped
│ │ ├─┬ @codingame/monaco-vscode-php-default-extension@7.1.1
│ │ │ └── vscode@npm:@codingame/monaco-vscode-api@7.1.1 deduped
│ │ ├─┬ @codingame/monaco-vscode-powershell-default-extension@7.1.1
│ │ │ └── vscode@npm:@codingame/monaco-vscode-api@7.1.1 deduped
│ │ ├─┬ @codingame/monaco-vscode-preferences-service-override@7.1.1
│ │ │ └── vscode@npm:@codingame/monaco-vscode-api@7.1.1 deduped
│ │ ├─┬ @codingame/monaco-vscode-python-default-extension@7.1.1
│ │ │ └── vscode@npm:@codingame/monaco-vscode-api@7.1.1 deduped
│ │ ├─┬ @codingame/monaco-vscode-quickaccess-service-override@7.1.1
│ │ │ └── vscode@npm:@codingame/monaco-vscode-api@7.1.1 deduped
│ │ ├─┬ @codingame/monaco-vscode-r-default-extension@7.1.1
│ │ │ └── vscode@npm:@codingame/monaco-vscode-api@7.1.1 deduped
│ │ ├─┬ @codingame/monaco-vscode-references-view-default-extension@7.1.1
│ │ │ └── vscode@npm:@codingame/monaco-vscode-api@7.1.1 deduped
│ │ ├─┬ @codingame/monaco-vscode-ruby-default-extension@7.1.1
│ │ │ └── vscode@npm:@codingame/monaco-vscode-api@7.1.1 deduped
│ │ ├─┬ @codingame/monaco-vscode-rust-default-extension@7.1.1
│ │ │ └── vscode@npm:@codingame/monaco-vscode-api@7.1.1 deduped
│ │ ├─┬ @codingame/monaco-vscode-scss-default-extension@7.1.1
│ │ │ └── vscode@npm:@codingame/monaco-vscode-api@7.1.1 deduped
│ │ ├─┬ @codingame/monaco-vscode-search-result-default-extension@7.1.1
│ │ │ └── vscode@npm:@codingame/monaco-vscode-api@7.1.1 deduped
│ │ ├─┬ @codingame/monaco-vscode-search-service-override@7.1.1
│ │ │ └── vscode@npm:@codingame/monaco-vscode-api@7.1.1 deduped
│ │ ├─┬ @codingame/monaco-vscode-shellscript-default-extension@7.1.1
│ │ │ └── vscode@npm:@codingame/monaco-vscode-api@7.1.1 deduped
│ │ ├─┬ @codingame/monaco-vscode-snippets-service-override@7.1.1
│ │ │ └── vscode@npm:@codingame/monaco-vscode-api@7.1.1 deduped
│ │ ├─┬ @codingame/monaco-vscode-sql-default-extension@7.1.1
│ │ │ └── vscode@npm:@codingame/monaco-vscode-api@7.1.1 deduped
│ │ ├─┬ @codingame/monaco-vscode-storage-service-override@7.1.1
│ │ │ └── vscode@npm:@codingame/monaco-vscode-api@7.1.1 deduped
│ │ ├─┬ @codingame/monaco-vscode-swift-default-extension@7.1.1
│ │ │ └── vscode@npm:@codingame/monaco-vscode-api@7.1.1 deduped
│ │ ├─┬ @codingame/monaco-vscode-terminal-service-override@7.1.1
│ │ │ └── vscode@npm:@codingame/monaco-vscode-api@7.1.1 deduped
│ │ ├─┬ @codingame/monaco-vscode-testing-service-override@7.1.1
│ │ │ └── vscode@npm:@codingame/monaco-vscode-api@7.1.1 deduped
│ │ ├─┬ @codingame/monaco-vscode-textmate-service-override@7.1.1
│ │ │ └── vscode@npm:@codingame/monaco-vscode-api@7.1.1 deduped
│ │ ├─┬ @codingame/monaco-vscode-theme-defaults-default-extension@7.1.1
│ │ │ └── vscode@npm:@codingame/monaco-vscode-api@7.1.1 deduped
│ │ ├─┬ @codingame/monaco-vscode-theme-service-override@7.1.1
│ │ │ └── vscode@npm:@codingame/monaco-vscode-api@7.1.1 deduped
│ │ ├─┬ @codingame/monaco-vscode-theme-seti-default-extension@7.1.1
│ │ │ └── vscode@npm:@codingame/monaco-vscode-api@7.1.1 deduped
│ │ ├─┬ @codingame/monaco-vscode-timeline-service-override@7.1.1
│ │ │ └── vscode@npm:@codingame/monaco-vscode-api@7.1.1 deduped
│ │ ├─┬ @codingame/monaco-vscode-typescript-basics-default-extension@7.1.1
│ │ │ └── vscode@npm:@codingame/monaco-vscode-api@7.1.1 deduped
│ │ ├─┬ @codingame/monaco-vscode-user-data-profile-service-override@7.1.1
│ │ │ └── vscode@npm:@codingame/monaco-vscode-api@7.1.1 deduped
│ │ ├─┬ @codingame/monaco-vscode-vb-default-extension@7.1.1
│ │ │ └── vscode@npm:@codingame/monaco-vscode-api@7.1.1 deduped
│ │ ├─┬ @codingame/monaco-vscode-view-status-bar-service-override@7.1.1
│ │ │ └── vscode@npm:@codingame/monaco-vscode-api@7.1.1 deduped
│ │ ├─┬ @codingame/monaco-vscode-views-service-override@7.1.1
│ │ │ ├─┬ @codingame/monaco-vscode-view-common-service-override@7.1.1
│ │ │ │ ├─┬ @codingame/monaco-vscode-bulk-edit-service-override@7.1.1
│ │ │ │ │ └── vscode@npm:@codingame/monaco-vscode-api@7.1.1 deduped
│ │ │ │ └── vscode@npm:@codingame/monaco-vscode-api@7.1.1 deduped
│ │ │ └── vscode@npm:@codingame/monaco-vscode-api@7.1.1 deduped
│ │ ├─┬ @codingame/monaco-vscode-workbench-service-override@7.1.1
│ │ │ ├─┬ @codingame/monaco-vscode-view-banner-service-override@7.1.1
│ │ │ │ └── vscode@npm:@codingame/monaco-vscode-api@7.1.1 deduped
│ │ │ ├─┬ @codingame/monaco-vscode-view-title-bar-service-override@7.1.1
│ │ │ │ └── vscode@npm:@codingame/monaco-vscode-api@7.1.1 deduped
│ │ │ └── vscode@npm:@codingame/monaco-vscode-api@7.1.1 deduped
│ │ ├─┬ @codingame/monaco-vscode-working-copy-service-override@7.1.1
│ │ │ └── vscode@npm:@codingame/monaco-vscode-api@7.1.1 deduped
│ │ ├─┬ @codingame/monaco-vscode-xml-default-extension@7.1.1
│ │ │ └── vscode@npm:@codingame/monaco-vscode-api@7.1.1 deduped
│ │ ├─┬ @codingame/monaco-vscode-yaml-default-extension@7.1.1
│ │ │ └── vscode@npm:@codingame/monaco-vscode-api@7.1.1 deduped
│ │ └── vscode@npm:@codingame/monaco-vscode-api@7.1.1 deduped
│ ├─┬ monaco-editor@npm:@codingame/monaco-vscode-editor-api@7.1.1
│ │ └── vscode@npm:@codingame/monaco-vscode-api@7.1.1 deduped
│ └─┬ vscode@npm:@codingame/monaco-vscode-api@7.1.1
│   ├─┬ @codingame/monaco-vscode-base-service-override@7.1.1
│   │ └── vscode@npm:@codingame/monaco-vscode-api@7.1.1 deduped
│   ├─┬ @codingame/monaco-vscode-environment-service-override@7.1.1
│   │ └── vscode@npm:@codingame/monaco-vscode-api@7.1.1 deduped
│   ├─┬ @codingame/monaco-vscode-host-service-override@7.1.1
│   │ └── vscode@npm:@codingame/monaco-vscode-api@7.1.1 deduped
│   └─┬ @codingame/monaco-vscode-layout-service-override@7.1.1
│     └── vscode@npm:@codingame/monaco-vscode-api@7.1.1 deduped
├─┬ @codingame/monaco-vscode-css-language-features-default-extension@8.0.2
│ └─┬ vscode@npm:@codingame/monaco-vscode-api@8.0.2
│   ├─┬ @codingame/monaco-vscode-base-service-override@8.0.2
│   │ └── vscode@npm:@codingame/monaco-vscode-api@8.0.2 deduped
│   ├─┬ @codingame/monaco-vscode-environment-service-override@8.0.2
│   │ └── vscode@npm:@codingame/monaco-vscode-api@8.0.2 deduped
│   ├─┬ @codingame/monaco-vscode-extensions-service-override@8.0.2
│   │ └── vscode@npm:@codingame/monaco-vscode-api@8.0.2 deduped
│   ├─┬ @codingame/monaco-vscode-files-service-override@8.0.2
│   │ └── vscode@npm:@codingame/monaco-vscode-api@8.0.2 deduped
│   ├─┬ @codingame/monaco-vscode-host-service-override@8.0.2
│   │ └── vscode@npm:@codingame/monaco-vscode-api@8.0.2 deduped
│   ├─┬ @codingame/monaco-vscode-layout-service-override@8.0.2
│   │ └── vscode@npm:@codingame/monaco-vscode-api@8.0.2 deduped
│   └─┬ @codingame/monaco-vscode-quickaccess-service-override@8.0.2
│     └── vscode@npm:@codingame/monaco-vscode-api@8.0.2 deduped
├─┬ @codingame/monaco-vscode-html-language-features-default-extension@8.0.2
│ └─┬ vscode@npm:@codingame/monaco-vscode-api@8.0.2
│   ├─┬ @codingame/monaco-vscode-base-service-override@8.0.2
│   │ └── vscode@npm:@codingame/monaco-vscode-api@8.0.2 deduped
│   ├─┬ @codingame/monaco-vscode-environment-service-override@8.0.2
│   │ └── vscode@npm:@codingame/monaco-vscode-api@8.0.2 deduped
│   ├─┬ @codingame/monaco-vscode-extensions-service-override@8.0.2
│   │ └── vscode@npm:@codingame/monaco-vscode-api@8.0.2 deduped
│   ├─┬ @codingame/monaco-vscode-files-service-override@8.0.2
│   │ └── vscode@npm:@codingame/monaco-vscode-api@8.0.2 deduped
│   ├─┬ @codingame/monaco-vscode-host-service-override@8.0.2
│   │ └── vscode@npm:@codingame/monaco-vscode-api@8.0.2 deduped
│   ├─┬ @codingame/monaco-vscode-layout-service-override@8.0.2
│   │ └── vscode@npm:@codingame/monaco-vscode-api@8.0.2 deduped
│   └─┬ @codingame/monaco-vscode-quickaccess-service-override@8.0.2
│     └── vscode@npm:@codingame/monaco-vscode-api@8.0.2 deduped
└─┬ @codingame/monaco-vscode-typescript-language-features-default-extension@8.0.2
  └─┬ vscode@npm:@codingame/monaco-vscode-api@8.0.2
    ├─┬ @codingame/monaco-vscode-base-service-override@8.0.2
    │ └── vscode@npm:@codingame/monaco-vscode-api@8.0.2 deduped
    ├─┬ @codingame/monaco-vscode-environment-service-override@8.0.2
    │ └── vscode@npm:@codingame/monaco-vscode-api@8.0.2 deduped
    ├─┬ @codingame/monaco-vscode-extensions-service-override@8.0.2
    │ └── vscode@npm:@codingame/monaco-vscode-api@8.0.2 deduped
    ├─┬ @codingame/monaco-vscode-files-service-override@8.0.2
    │ └── vscode@npm:@codingame/monaco-vscode-api@8.0.2 deduped
    ├─┬ @codingame/monaco-vscode-host-service-override@8.0.2
    │ └── vscode@npm:@codingame/monaco-vscode-api@8.0.2 deduped
    ├─┬ @codingame/monaco-vscode-layout-service-override@8.0.2
    │ └── vscode@npm:@codingame/monaco-vscode-api@8.0.2 deduped
    └─┬ @codingame/monaco-vscode-quickaccess-service-override@8.0.2
      └── vscode@npm:@codingame/monaco-vscode-api@8.0.2 deduped`

> The version is ^8.0.0 which accepts any minor version though

Thanks. I was not sure if npm would allow minor changes in its dependencies. I though it would lock it to the version it has been published with.

Thank you!

**Comment by CGNonofr** [comment]
OK you have duplicated versions of @codingame/monaco-vscode-api (8.0.2 and 7.1.1) because monaco-editor-wrapper wasn't updated yet to the v8 of monaco-vscode-api

**Comment by jpdery** [comment]
Fixed with version 9.0. Thanks!

**Comment by CGNonofr** [comment]
Nothing was fixed with version 9, it just forced your npm to update all packages to the last version
